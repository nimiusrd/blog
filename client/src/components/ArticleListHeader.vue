<template>
    <li class="item">
        <div class="title">
            TITLE
        </div>
        <div class="author">
            AUTHOR
        </div>
        <div class="posted-at">
            POSTED AT
        </div>
        <div class="updated-at">
            UPDATED AT
        </div>
    </li>
</template>

<script lang="ts">
import Vue from 'vue'

const ArticleListHeader = Vue.extend({})

export default ArticleListHeader
</script>

<style scoped>
.item {
    display: grid;
    font-size: 20px;
    font-weight: 600;
    grid-template:
        "title author posted-at updated-at"
        / 3fr 1fr 1fr 1fr;
    text-align: center;
}

.title {
    border: 1px black solid;
    grid-area: title;
    padding: 16px;
}

.author {
    border: 1px black solid;
    grid-area: author;
    padding: 16px;

}

.posted-at {
    border: 1px black solid;
    grid-area: posted-at;
    padding: 16px;
}

.updated-at {
    border: 1px black solid;
    grid-area: updated-at;
    padding: 16px;
}
</style>
