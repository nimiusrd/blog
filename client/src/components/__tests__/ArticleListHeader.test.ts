import ArticleListHeader from '@/components/ArticleListHeader.vue'
import {shallowMount} from '@vue/test-utils'

test('snapshot', () => {
    const wrapper = shallowMount(ArticleListHeader)

    expect(wrapper.html()).toMatchSnapshot()
})
