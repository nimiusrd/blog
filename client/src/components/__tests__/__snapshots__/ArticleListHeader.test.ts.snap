// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
"<li class=\\"item\\"><div class=\\"title\\">
        TITLE
    </div> <div class=\\"author\\">
        AUTHOR
    </div> <div class=\\"posted-at\\">
        POSTED AT
    </div> <div class=\\"updated-at\\">
        UPDATED AT
    </div></li>"
`;
